Adding Webscraper presentation

WillQualls opened the pull request:
[] I have added all the dependencies used in my pull request to requirements.txt.
[] I have updated the requirements.txt if any dependency versions were changed. 
I Included the presentation and any required images in the images folder
I updated the _quarto.yml to include my name and the presentation

jun-yan commented:
Could you clean up the Topic folder? There appears to be another _webscraping.qmd file.

WillQualls commented:
I believe I fixed the issue, I deleted the Topic folder, that was just where I was working before putting it in the book. I also deleted another erroneous folder "webscraping_files", which looked to contain mostly garbage to me.
